sdk/storage/azure-storage-file-datalake/README.md

```diff
@@ -25,17 +25,20 @@ pip install azure-storage-file-datalake --pre
 
 ### Create a storage account
 If you wish to create a new storage account, you can use the
-[Azure Portal](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-the-azure-portal),
-[Azure PowerShell](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-powershell),
-or [Azure CLI](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-quickstart-create-account#create-an-account-using-azure-cli):
+[Azure Portal](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal),
+[Azure PowerShell](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-powershell),
+or [Azure CLI](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-cli):
 
 ```bash
 # Create a new resource group to hold the storage account -
 # if using an existing resource group, skip this step
 az group create --name my-resource-group --location westus2
 
+# Install the extension “Storage-Preview”
+az extension add --name storage-preview
```

Comment:
what is this used for?

Reply:
According to this, [MicrosoftDocs/azure-docs#24481](https://github.com/MicrosoftDocs/azure-docs/issues/24481), CLI extension is required.

Reply:
I added a comment on that issue, it looks like MicrosoftDoc needs to be updated.